src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -818,6 +821,36 @@ void setBinding(DependencyProperty property, string path)
 			ContentTemplateRoot = textBlock;
 		}
 
+		private void SetImplicitContent()
```

Comment:
the same thing should be done for `ContentTemplate` as well.

Reply:
That would be another PR.